src/module.ts

```diff
@@ -233,92 +226,96 @@ export const enum FeatureFlags {
-  Try = 49 /* _BinaryenTryId */,
-  Throw = 50 /* _BinaryenThrowId */,
-  Rethrow = 51 /* _BinaryenRethrowId */,
-  TupleMake = 52 /* _BinaryenTupleMakeId */,
-  TupleExtract = 53 /* _BinaryenTupleExtractId */,
-  I31New = 54 /* _BinaryenI31NewId */,
-  I31Get = 55 /* _BinaryenI31GetId */,
-  CallRef = 56 /* _BinaryenCallRefId */,
-  RefTest = 57 /* _BinaryenRefTestId */,
-  RefCast = 58 /* _BinaryenRefCastId */,
-  BrOn = 59 /* _BinaryenBrOnId */,
-  StructNew = 60 /* _BinaryenStructNewId */,
-  StructGet = 61 /* _BinaryenStructGetId */,
-  StructSet = 62 /* _BinaryenStructSetId */,
-  ArrayNew = 63 /* _BinaryenArrayNewId */,
-  ArrayNewSeg = 64 /* TODO_BinaryenArraySegId */,
-  ArrayNewFixed = 65 /* _BinaryenArrayNewFixedId */,
-  ArrayGet = 66 /* _BinaryenArrayGetId */,
-  ArraySet = 67 /* _BinaryenArraySetId */,
-  ArrayLen = 68 /* _BinaryenArrayLenId */,
-  ArrayCopy = 69 /* _BinaryenArrayCopyId */,
-  ArrayFill = 70 /* _BinaryenArrayFillId */,
-  ArrayInit = 71 /* _BinaryenArrayInitId */,
-  RefAs = 72 /* _BinaryenRefAsId */,
-  StringNew = 73 /* _BinaryenStringNewId */,
-  StringConst = 74 /* _BinaryenStringConstId */,
-  StringMeasure = 75 /* _BinaryenStringMeasureId */,
-  StringEncode = 76 /* _BinaryenStringEncodeId */,
-  StringConcat = 77 /* _BinaryenStringConcatId */,
-  StringEq = 78 /* _BinaryenStringEqId */,
-  StringAs = 79 /* _BinaryenStringAsId */,
-  StringWTF8Advance = 80 /* _BinaryenStringWTF8AdvanceId */,
-  StringWTF16Get = 81 /* _BinaryenStringWTF16GetId */,
-  StringIterNext = 82 /* _BinaryenStringIterNextId */,
-  StringIterMove = 83 /* _BinaryenStringIterMoveId */,
-  StringSliceWTF = 84 /* _BinaryenStringSliceWTFId */,
-  StringSliceIter = 85 /* _BinaryenStringSliceIterId */
+  Invalid = 0,
+  Block,
+  If,
+  Loop,
+  Break,
+  Switch,
+  Call,
+  CallIndirect,
+  LocalGet,
+  LocalSet,
+  GlobalGet,
+  GlobalSet,
+  Load,
+  Store,
+  Const,
+  Unary,
+  Binary,
+  Select,
+  Drop,
+  Return,
+  MemorySize,
+  MemoryGrow,
+  Nop,
+  Unreachable,
+  AtomicRMW,
+  AtomicCmpxchg,
+  AtomicWait,
+  AtomicNotify,
+  AtomicFence,
+  SIMDExtract,
+  SIMDReplace,
+  SIMDShuffle,
+  SIMDTernary,
+  SIMDShift,
+  SIMDLoad,
+  SIMDLoadStoreLane,
+  MemoryInit,
+  DataDrop,
+  MemoryCopy,
+  MemoryFill,
+  Pop,
+  RefNull,
+  RefIsNull,
+  RefFunc,
+  RefEq,
+  TableGet,
+  TableSet,
+  TableSize,
+  TableGrow,
+  TableFill,
+  Try,
+  Throw,
+  Rethrow,
+  TupleMake,
+  TupleExtract,
+  RefI31,
+  I31Get,
+  CallRef,
+  RefTest,
+  RefCast,
+  BrOn,
+  StructNew,
+  StructGet,
+  StructSet,
+  ArrayNew,
+  ArrayNewData,
+  ArrayNewElem,
+  ArrayNewFixed,
+  ArrayGet,
+  ArraySet,
+  ArrayLen,
+  ArrayCopy,
+  ArrayFill,
+  ArrayInitData,
+  ArrayInitElem,
+  RefAs,
+  StringNew,
+  StringConst,
+  StringMeasure,
+  StringEncode,
+  StringConcat,
+  StringEq,
+  StringAs,
+  StringWTF8Advance,
+  StringWTF16Get,
+  StringIterNext,
+  StringIterMove,
+  StringSliceWTF,
+  StringSliceIter,
+  NumExpressionIds
```

Comment:
Hmm, why autogen constant comments are gone?

Reply:
Where is the autogen script?

Reply:
just run `node scripts/update-constants.js`